core/includes/common.inc

```diff
@@ -8483,6 +8483,326 @@ function _backdrop_flush_css_js() {
   state_set('css_js_query_string', base_convert(REQUEST_TIME, 10, 36));
 }
 
+/**
```

Comment:
Might be useful to have a `@defgroup` around all these functions in common.inc since it's a long file. Help to group all the related functions. Something like:

```
/**
 * @defgroup cache_tags
 * @{
 * Cache Tags
 */
```